tests/test_value_source_validator.py

```diff
@@ -12,6 +12,8 @@
         {"source": "metadata", "identifier": "submitted_at"},
         {"source": "response_metadata", "identifier": "submitted_at"},
         {"source": "calculated_summary", "identifier": "total-turnover"},
+        {"source": "progress", "identifier": "people"},
+        {"source": "progress", "selector": "group", "identifier": "people"},
```

Comment:
Not valid according to the schema. Selector is required and "group" isn't a valid selector.